Source/WebKit/UIProcess/UserMediaPermissionRequestManagerProxy.cpp

```diff
@@ -562,8 +562,8 @@ String UserMediaPermissionRequestManagerProxy::ephemeralDeviceHashSaltForFrame(W
     static constexpr unsigned hashSaltSize { 48 };
     static constexpr unsigned randomDataSize { hashSaltSize / 16 };
 
-    uint64_t randomData[randomDataSize];
-    cryptographicallyRandomValues(reinterpret_cast<unsigned char*>(randomData), sizeof(randomData));
+    std::array<uint64_t, randomDataSize> randomData;
+    cryptographicallyRandomValues({ reinterpret_cast<uint8_t*>(randomData.data()), sizeof(randomData) });
```

Comment:
This idiom comes up multiple times. The `std::as_writeable_bytes` function is for cases like this, but it uses `std:.byte`, not `std::uint8_t`. Maybe we should make our own version or even consider using `std::byte` some day.

Reply:
Ok, I'm thinking of adding this to Algorithms.h (until we can move to StdLibExtras.h):
```
template<typename T>
std::span<const uint8_t> asBytes(std::span<T> span)
{
    return { reinterpret_cast<const uint8_t*>(span.data()), span.size() * sizeof(T) };
}

template<typename T>
std::span<uint8_t> asWritableBytes(std::span<T> span)
{
    return { reinterpret_cast<uint8_t*>(span.data()), span.size() * sizeof(T) };
}
```